deriva/core/ermrest_catalog.py

```diff
@@ -40,20 +39,11 @@ def getCatalogConfig(self):
     def applyCatalogConfig(self, config):
         return config.apply(self)
 
-    def getCatalogSchema(self, force=False):
-        """Gets the catalog schema.
-        This function caches the schema and returns it on subsequent calls.
-        :param force: set True to ignore cached schema (default:False).
-        """
-        if self.catalog_schema and not force:
-            return self.catalog_schema
-
+    def getCatalogSchema(self):
         path = '/schema'
         r = self.get(path)
         resp = r.json()
```

Comment:
This actually seems out of order.  The raise_for_status should happen before r.json() is called, since the json decoding would fail if the response status_code was not 200 w/ the expected json payload...